Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -321,6 +322,92 @@ then the same range must be passed to Cilium via the ``global.nodePort.range``
 option, for example, as ``--set global.nodePort.range="10000\,32767"`` for a
 range of ``10000-32767``. The default Kubernetes NodePort range is ``30000-32767``.
 
+Container hostPort support
+**************************
+
+Although not part of kube-proxy, Cilium's BPF kube-proxy replacement also
+natively supports ``hostPort`` service mapping without having to use the
+Helm CNI chaining option of ``global.cni.chainingMode=portmap``.
+
+By specifying ``global.kubeProxyReplacement=strict`` or ``global.kubeProxyReplacement=probe``
+the native hostPort support is automatically enabled and therefore no further
+action is required. Otherwise ``global.hostPort.enabled=true`` can be used to
+enable the setting.
+
+An example deployment in a kube-proxy-free environment therefore is the same
+as in the earlier getting started deployment:
+
+.. parsed-literal::
+
+    helm install cilium |CHART_RELEASE| \\
+        --namespace kube-system \\
+        --set global.kubeProxyReplacement=strict \\
+        --set global.k8sServiceHost=API_SERVER_IP \\
+        --set global.k8sServicePort=API_SERVER_PORT
+
+The following modified example yaml from the setup validation with an additional
+``hostPort: 8080`` parameter can be used to check the mapping:
+
+.. parsed-literal::
+
+    apiVersion: apps/v1
```

Comment:
Nit: `kubectl run nginx --generator=run-pod/v1 --image=nginx --replicas=1 --port=80 --hostport=8080`?

Reply:
I was referring to the example yaml from earlier in the same doc, so I kept it as is but with the added `hostPort`.